pkg/backup_servers/util.go

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright (c) 2021 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+package backup_servers
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	orgv1 "github.com/eclipse-che/che-operator/pkg/apis/org/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// getResticRepoPassword checks if the password for restic repository is specified and retrieves it.
+// It doesn't check the password correctness.
+// Returns:
+//  - password or empty string if password is not set.
```

Comment:
```suggestion
//  - password or empty string if password is not set
```
just to be consistent